docs/pages/management/guides/elastic-stack.mdx

```diff
@@ -0,0 +1,729 @@
+requests to the Elasticsearch API. 
+
+In Kibana, find the hamburger menu on the upper left and click "Management",
+then "Users" > "Create user". Enter `teleport` for the "Username" and provide a
+secure password.
+
+Assign the user the `teleport-plugin` role we defined earlier.
+
+### Prepare TLS credentials for Logstash
+
+Later in this guide, your Logstash pipeline will use an HTTP input to receive
+audit events from the Teleport Event Handler plugin.
+
+Logstash's HTTP input can only sign certificates with a private key that uses
+the unencrypted PKCS #8 format. When you ran `teleport-event-handler configure`
+earlier, the command generated an encrypted RSA key. We will convert this key to
+PKCS #8.
+
+You will need a password to decrypt the RSA key. To retrieve this, execute the
+following command in the directory where you ran `teleport-event-handler
+configure`:
+
+```code
+$ cat fluent.conf | grep passphrase
+private_key_passphrase "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+```
+
+Convert the encrypted RSA key to an unencrypted PKCS #8 key. The command will
+prompt your for the password you retrieved: 
+
+```code
+$ openssl pkcs8 -topk8 -in server.key -nocrypt -out pkcs8.key
+```
+
+Enable Logstash to read the new key, plus the CA and certificate we generated
+earlier:
+
+```code
+$ chmod +r pkcs8.key ca.crt server.crt
+```
+
+### Define an index template
+
+When the Event Handler plugin sends audit events to Logstash, Logstash needs to
+know how to parse these events to forward them to Elasticsearch. You can define
+this logic using an index template, which Elasticsearch uses to construct an
+index for data it receives.
+
+Create a file called `audit-events.json` with the following content:
+
+```json
+{
+  "index_patterns": ["audit-events-*"],
+  "template": {
+    "settings": {},
+    "mappings": {
+      "dynamic":"true"
+    }
+  }
+}
+```
+
+This index template modifies any index with the pattern `audit-events-*`.
+Because it includes the `"dynamic": "true"` setting, it instructs Elasticsearch
+to define index fields dynamically based on the events it receives. This is
+useful for Teleport audit events, which use a variety of fields depending on
+the event type.
+
+### Define a Logstash pipeline
+
+On the host where you are running Logstash, create a configuration file that
+defines a Logstash pipeline. This pipeline will receive logs from port `9601`
+and forward them to Elasticsearch.
+
+On the host running Logstash, create a file called
+`/etc/logstash/conf.d/teleport-audit.conf` with the following content:
+ 
+```ruby
+input {
+  http {
+    port => 9601
+    ssl =>  true
+    ssl_certificate => "/home/server.crt"
+    ssl_key =>  "/home/pkcs8.key"
+    ssl_certificate_authorities => [
+      "/home/ca.crt"
+    ]
+    ssl_verify_mode => "force_peer"
+  }
+}
+output {
+  elasticsearch {
+    user => "teleport"
+    password => "ELASTICSEARCH_PASSPHRASE"
```

Comment:
Should we include a gotcha warning for those not paying attention to change the value here?